app/javascript/dashboard/components/widgets/WootWriter/Editor.vue

```diff
@@ -616,6 +605,44 @@ export default {
     onFocus() {
       this.$emit('focus');
     },
+    insertContentIntoEditor(content, defaultFrom = 0) {
+      const from = defaultFrom || this.editorView.state.selection.from || 0;
+      let node = new MessageMarkdownTransformer(messageSchema).parse(content);
+
+      this.insertNodeIntoEditor(node, from, undefined, content);
+    },
+    insertNodeIntoEditor(node, from = 0, to = 0, content = '') {
```

Comment:
@nithindavid Is it possible to move this method to the helper file? `app/javascript/dashboard/helper/messageEditorHelper.js`